moveit_core/planning_interface/include/moveit/planning_interface/planning_response.h

```diff
@@ -37,23 +37,39 @@
 #pragma once
 
 #include <moveit/robot_trajectory/robot_trajectory.h>
+#include <moveit/utils/moveit_error_code.h>
 #include <moveit_msgs/msg/move_it_error_codes.hpp>
 #include <moveit_msgs/msg/motion_plan_response.hpp>
 #include <moveit_msgs/msg/motion_plan_detailed_response.hpp>
 
 namespace planning_interface
 {
+/// \brief Response to a planning query
 struct MotionPlanResponse
 {
-  MotionPlanResponse() : planning_time_(0.0)
+  // Constructor
```

Comment:
```suggestion
  /** \brief Constructor */
```

Reply:
You can also just drop this comment entirely. A comment telling us the next line is a constructor is not that helpful

Reply:
:thinking: ... you're right :sweat_smile: